### PR TITLE
Feature/handle live query diff

### PR DESCRIPTION
**Pull request by Atoo35**
@phughk i am pretty sure i have done something horrible in here. guide me please

This PR resolves issue #81

### CODE REVIEW

**Review by phughk — COMMENTED** [review]
Left a comment, but would be good to get @timpratim s eyes on this as well. Ex the deserialisation. I am creating a ticket to move it out of gorilla and test. Otherwise I think it's ok, thanks for fixing!

**Comment by Atoo35** [comment]
@phughk i have fixed like you suggested. Although not sure why the test is failing here on github, being a developer "it worked just fine on my machine" !!

**Comment by ilkerkorkut** [comment]
> @phughk i have fixed like you suggested. Although not sure why the test is failing here on github, being a developer "it worked just fine on my machine" !!

Most probably it's related with that issue -> https://github.com/surrealdb/surrealdb.go/issues/75

**Comment by Atoo35** [comment]
thanks @ElecTwix the work was done already, was just stuck in the test pipeline here. @phughk is this fine to be merged now?

File: db.go [hunk 1]
```diff
@@ -63,8 +63,8 @@ func (db *DB) Authenticate(token string) (interface{}, error) {
 
 // --------------------------------------------------
 
-func (db *DB) Live(table string) (string, error) {
-	id, err := db.send("live", table)
+func (db *DB) Live(table string, updateType LiveQueryUpdateType) (string, error) {
```
Comment (phughk) [inline db.go:66]: Could we remove `LiveQueryUpdateType` and replace it with just `bool` please? It looks like that is how the protocol is implemented.
https://surrealdb.com/docs/integration/websocket/text#live

**Review by phughk — COMMENTED** [review]
Happy to approve once the param is changed.
Thanks for this @Atoo35

**Comment by Atoo35** [comment]
@phughk made the change as requested. Thanks a lot !

**Review by phughk — APPROVED** [review]
Lgtm thanks @Atoo35 !

**Pull request merged**